.github/workflows/full.yml

```diff
@@ -0,0 +1,578 @@
+name: full pipeline
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
```

Comment:
If `pull_request` is empty why do we need it?

Reply:
If it is empty, it will match all branches, but without it won't match pull requests.

Reply:
Okey, thanks for the clarification.